types.go

```diff
@@ -24,7 +25,7 @@ type ObjectMeta struct {
 	Namespace string `yaml:"namespace,omitempty"`
 }
 
-// DeploymentStatus is the most recently observed status of the Deployment.
+// DeploymentStatus is the most recently observed status of the Deployment / StatefullSet.
```

Comment:
StatefullSet :)

Reply:
Cheers - good spot - I'll push the update :)